### PR TITLE
new line for username and password

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ SonataAdminBundle - The missing Symfony2 Admin Generator
 Demo
 ----
 
-The demo website can be found in http://demo.sonata-project.org/admin/dashboard
+The demo website can be found in http://demo.sonata-project.org/admin/dashboard
```
Comment (Soullivaneuh) [inline README.md]: Trailling whitespaces

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ SonataAdminBundle - The missing Symfony2 Admin Generator
 Demo
 ----
 
-The demo website can be found in http://demo.sonata-project.org/admin/dashboard
+The demo website can be found in http://demo.sonata-project.org/admin/dashboard    
 **Username and password is:** ```admin```
```
Comment (Soullivaneuh) [inline README.md]: Please add a blank line before.

File: README.md (hunk 1 above)
Comment (OskarStark) [inline README.md]: In markdown 4 trailing spaces is a line Break

Comment (Soullivaneuh) [inline README.md]: What do you mean?

Can you simply add a line separation between this and the "Username and password is:"?

This is more clear and will avoid problem with auto trailling whitespaces remover on some IDE.

**Comment by OskarStark** [comment]
@Soullivaneuh i did it, but this is not what i wanted to do:

my result before:
bildschirmfoto 2015-07-09 um 12 29 53


my result after the change you wish:
bildschirmfoto 2015-07-09 um 12 30 19

thats what i said, in markdown to make new line you have to add 4 spaces, to make a new paragraph you have to add a blank line. hope you understand now, what i meant

**Comment by Soullivaneuh** [comment]
I especially want the second result. :-)

**Comment by Soullivaneuh** [comment]
Can you please squash your commits?

**Comment by OskarStark** [comment]
done @Soullivaneuh

**Comment by OskarStark** [comment]
Thank you!

**Pull request merged**